src/main/java/Duke.java

```diff
@@ -1,10 +1,411 @@
+                tasks.add(new Deadline(task, dueDate));
+                int taskIndex = tasks.size() - 1;
+                printTaskAcknowledgement(tasks.get(taskIndex));
+            }
+        } catch (StringIndexOutOfBoundsException e) {
+            System.out.println(LINES + MISSING_IN_DEADLINE_MESSAGE + LINES);
+        }
+    }
+    private static void addEvent(String input) {
+        try {
+            if (input.trim().length() == 5) {
+                System.out.println(LINES + PLEASE_ADD_A_TASK_DESCRIPTION + LINES);
+            } else if (!input.contains("/at")) {
+                System.out.println(LINES + MISSING_AT_MESSAGE + LINES);
+            } else {
+                int slashIndex = input.indexOf("/at");
+                String task = input.substring(6, slashIndex).trim();
+                String timeRange = input.substring(slashIndex + 3).trim();
+                tasks.add(new Event(task, timeRange));
+                int taskIndex = tasks.size() - 1;
+                printTaskAcknowledgement(tasks.get(taskIndex));
+            }
+        } catch (StringIndexOutOfBoundsException e) {
+            System.out.println(LINES + MISSING_IN_EVENT_MESSAGE + LINES);
+        }
+    }
+    private static void printTaskAcknowledgement(Task task) {
+        System.out.println(LINES + ADD_MESSAGE + task);
+        System.out.println(" Now you have " + tasks.size() + " tasks in the list." +
+                System.lineSeparator() + LINES);
+    }
+
+    public static void appendDatabase(String type, String input, String isDone) {
+        //append new tasks to database
+        try {
+            FileWriter fw = new FileWriter(FILEPATH, true);
+            switch (type) {
+            case "T":
+                fw.write(type + " | " + isDone + " | " + input + System.lineSeparator());
+                break;
+            case "D":
+                String[] deadlineSplit = input.split("/by", 2);
+                fw.write(type + " | " + isDone + " | " + deadlineSplit[0] + " | " +
+                        deadlineSplit[1] + System.lineSeparator());
+                break;
+            case "E":
+                String[] eventSplit = input.split("/at", 2);
+                fw.write(type + " | " + isDone + " | " + eventSplit[0] + " | " + eventSplit[1]
+                        + System.lineSeparator());
+                break;
+            default:
+                break;
+            }
+            fw.close();
+        } catch (IOException e) {
+            System.out.println(LINES + NO_FILE_MESSAGE + LINES);
+        }
+    }
+
+    public static void clearDatabase() {
+        try {
+            FileWriter fw = new FileWriter(FILEPATH);
+            fw.write("");
+            fw.close();
+        } catch (IOException e) {
+            System.out.println(LINES + NO_FILE_MESSAGE + LINES);
+        }
+    }
+
+    public static void updateDatabase() {
+        //update after mark done.
+        //clear file then append from list
```

Comment:
You may want to move the comments outside of the method (just before the start of the method) and just describe at a high level what the method does if you wish to. Do not have to include implementation details in the comments unless the logic is extremely complex.